finality-aleph/src/sync/handler.rs

```diff
@@ -465,12 +548,12 @@ mod tests {
             .expect("importing in order");
         let justification = MockJustification::for_header(header);
         let peer = rand::random();
-        assert!(
+        assert!(matches!(
             handler
                 .handle_justification(justification.clone().into_unverified(), Some(peer))
-                .expect("correct justification")
-                == Some(justification.id())
-        );
+                .expect("correct justification"),
+            Some(id) if id == justification.id()
```

Comment:
You can now go back to the equality check, it looks less mysterious.

Reply:
fixed